abb_irb2400_support/package.xml

```diff
@@ -4,6 +4,8 @@
   <description>
     <p>
       ROS-Industrial support for the ABB IRB 2400 (and variants).
+
+      The irb2400_12_155 should be used for the IRB 2400 20kg 1.55m variant.
```

Comment:
The first paragraph of the `description` element is used for the *short description* of the (Debian) packages by Bloom.

Suggestion: add a separate paragraph (`<p>`) for this ([example](https://github.com/ros-industrial/fanuc/blob/798c4bf5d1c472157964f131dda157ea989d9ed6/fanuc_lrmate200ic_support/package.xml#L10-L15)).